test/utils/load-rule.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const path = require('path');
+const decomment = require('decomment');
+
+/**
+ * Loads a rule, optionally with stubs
+ *
+ * @param {string} ruleFileName - file name of rule to load
+ * @param {object} globals - the global context the rule is executed within
+ * @param {object} stubs - modules to override when required by the rule
+ **/
+module.exports = function (ruleFileName, globals, stubs) {
+  globals = globals || {};
+  stubs = stubs || {};
+  
+  const fileName = path.join(__dirname, '../../src/rules', ruleFileName);
+
+  const file = fs.readFileSync(fileName, 'utf8');
+
+  const code =  decomment(file).trim();
+
+  return compile(code, globals, stubs);
+}
+
+function compile(code, globals, stubs) {
+  function fakeRequire (moduleName) {
+    if (stubs[moduleName]) {
+      return stubs[moduleName];
+    }
+
+    return require(moduleName);
+  }
+
+  const globalObj = Object.assign({}, { require: fakeRequire }, globals);
```

Comment:
https://github.com/auth0/auth0-authz-rules-api/blob/master/index.js might be useful in helping define the "magic globals" that are used in Rules.